js/pages/tablespage.ts

```diff
@@ -163,6 +163,14 @@ export class TablesPage extends PageBase implements ICurrentTableProvider {
 
         let viewportLandscapeWidth = 640;
         const currentWidth = $("body").width();
+        if (currentWidth >= 667) {
+            viewportLandscapeWidth = 667;
+        }
+
+        if (currentWidth >= 736) {
```

Comment:
Take in the account that user should open phone in the landscape orientation, and as such width and height change their values